cmd/up.go

```diff
@@ -57,9 +59,15 @@ func Up() *cobra.Command {
 			stopChannel := make(chan os.Signal, 1)
 			signal.Notify(stopChannel, os.Interrupt)
 			log.Debugf("%s ready, waiting for stop signal to shut down", fullname)
-			<-stopChannel
-			fmt.Println()
-			return nil
+			for {
+				select {
+				case <-stopChannel:
+					fmt.Println()
+					return nil
+				case <-disconnectChannel:
+					return fmt.Errorf("syncthing is not available, disconnecting")
```

Comment:
@rlamana this is the message that's displayed when the CLI gets disconnected, what do you think?